ci_framework/roles/artifacts/tasks/edpm.yml

```diff
@@ -77,8 +77,7 @@
           sudo test -d /var/lib/openstack && sudo cp -a /var/lib/openstack /tmp/{{ host_ip }}
           sudo test -d /var/lib/config-data && sudo cp -a /var/lib/config-data /tmp/{{ host_ip }}
           sudo test -d /var/lib/cloud && sudo cp -a /var/lib/cloud /tmp/{{ host_ip }}
-          sudo test -d /etc/nftables && sudo cp -a /etc/nftables /tmp/{{ host_ip }}
-          sudo test -d /etc/os-net-config && sudo cp -a /etc/os-net-config /tmp/{{ host_ip }}
+          sudo test -d /etc && sudo cp -a /etc /tmp/{{ host_ip }}
```

Comment:
I see potential issues with the recursion depth, some directories there have symlink to other places and may have a really deep recursion. Such recursions are usually really bad when it comes to logserver sync, unfortunately.

Would it be possible to get a listing of directories of interest?

Reply:
I know what I need today, the /etc/ssh/ssh_know_hosts file, but I cannot predict what I will need in my next CI troubleshooting. I agree about the recursion issue though. What about `cp -RP /etc` ? That would follow non of the symlink